applications/GeoMechanicsApplication/tests/cpp_tests/custom_utilities/test_stress_strain_utitlities.cpp

```diff
@@ -190,4 +192,62 @@ KRATOS_TEST_CASE_IN_SUITE(CheckCalculateStrains, KratosGeoMechanicsFastSuiteWith
         KRATOS_EXPECT_VECTOR_NEAR(strains[i], expected_strains[i], 1.E-6)
 }
 
+KRATOS_TEST_CASE_IN_SUITE(CheckCalculatePrincipalStresses, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    Vector cauchy_stresses = ZeroVector(6);
+    cauchy_stresses <<= 80.0, 50.0, 20.0, 40.0, 35.0, 45.0;
+
+    Vector principal_stresses;
+    Matrix eigenvectors_matrix;
+    StressStrainUtilities::CalculatePrincipalStresses(cauchy_stresses, principal_stresses, eigenvectors_matrix);
+
+    Vector expected_solution = ZeroVector(3);
+    expected_solution <<= 135.736961146391, 22.5224297324582, -8.25939087884923;
+
+    constexpr double tolerance{1.0e-12};
+
+    for (unsigned int i = 0; i < principal_stresses.size(); ++i) {
+        KRATOS_EXPECT_NEAR(principal_stresses(i), expected_solution(i), tolerance);
+    }
+}
+
+KRATOS_TEST_CASE_IN_SUITE(CheckCalculateRotationMatrix2D, KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Set
+    auto   plane_strain  = std::make_unique<PlaneStrain>();
+    Vector stress_vector = ZeroVector(plane_strain->GetStrainSize());
+    stress_vector <<= -6.0, -7.0, -14.0, 1.0;
```

Comment:
I think using class `PlaneStrain` is overkill here. When you fill `stress_vector`, you know that pass it four values.
```suggestion
    Vector stress_vector(4);
    stress_vector <<= -6.0, -7.0, -14.0, 1.0;
```